.travis.yml

```diff
@@ -6,10 +6,9 @@ before_install:
   - pod repo update
 
 before_script:
-  - wget https://github.com/realm/SwiftLint/releases/download/0.29.3/portable_swiftlint.zip -O /tmp/swiftlint.zip
-  - unzip /tmp/swiftlint.zip -d ${PWD}/bin
-  - export PATH=$PATH:${PWD}/bin
+  - wget https://github.com/realm/SwiftLint/releases/download/0.29.3/portable_swiftlint.zip
```

Comment:
In the future it might be useful to use Travis's [caching](https://docs.travis-ci.com/user/caching/) mechanisms to download to a common `bin` folder, say, and modify this check to only download/extract if a specific version doesn't already exist in the cache.